ui/app/AppLayouts/Chat/ChatColumn.qml

```diff
@@ -90,8 +91,36 @@ StackLayout {
             id: profilePopup
         }
 
+
+        EmojiReactions {
+            id: reactionModel
+        }
+
         PopupMenu {
             id: messageContextMenu
+            width: emojiRow.width
+
+            Row {
```

Comment:
Very cool! I did not know you could add items other than Actions on the PopupMenu!

Reply:
Me neither haha. I'm just happy it worked